presto-main/src/main/java/com/facebook/presto/event/query/QueryMonitor.java

```diff
@@ -176,6 +176,11 @@ public void queryCompletedEvent(QueryInfo queryInfo)
                                 tableFinishInfo.map(TableFinishInfo::isJsonLengthLimitExceeded)));
```

Comment:
What's the purpose of this commit? Maybe add some details to the commit message.